### PR TITLE
HomeView: Add home page

### PR DESCRIPTION
**Pull request by HWaruguru**
This is a work-in-progress for the home page component

Task: [T326870](https://phabricator.wikimedia.org/T326870)

### CODE REVIEW

**Review by nbarnabee — APPROVED** [review]
Looks good.

**Comment by nbarnabee** [comment]
In hindsight, I probably shouldn't review things right before bed.  We may not want to have a search bar in there at the moment, since a search option wasn't going to be part of the MVP.

**Comment by HWaruguru** [comment]
That's right let me work on it

File: plugins/vuetify.js [hunk 1]
```diff
@@ -1,13 +1,48 @@
 import "vuetify/styles";
 import { createVuetify } from "vuetify";
-import { aliases, mdi } from "vuetify/iconsets/midi";
+import '@mdi/font/css/materialdesignicons.css';
```
Comment (Damilare1) [inline plugins/vuetify.js:3]: Looks like we didn't get to use any icon from this import. I'd suggest we take it off for now.

**Review by Damilare1 — CHANGES_REQUESTED** [review]
Looks good, just a small suggestion and we'd be good to go.

**Review by Damilare1 — APPROVED** [review]

**Pull request merged**